presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -25,7 +25,8 @@
 public class PrestoSparkConfig
 {
     private boolean sparkPartitionCountAutoTuneEnabled = true;
-    private int initialSparkPartitionCount = 16;
+    private int maxSparkInputPartitionCountForAutoTune = 1000;
+    private int initialSparkPartitionCount = 100;
```

Comment:
Just curious, will this be the new default to replace 2000 in production? If not,  maybe keep it as 16 to make it easier for `PrestoQueryRunner`